Tests/test_PDB_ResidueDepth.py

```diff
@@ -45,39 +47,51 @@ def check_msms(self, prot_file, first_100_residues):
         self.assertEqual(res_chain, first_100_residues)
 
     def test_ResidueDepth_2XHE(self):
-        self.check_msms("PDB/2XHE.pdb", "HISMETSERLEULYSSERALAVALLYSTHRVALLEUTH"
-                                        "RASNSERLEUARGSERVALALAASPGLYGLYASPTRPL"
-                                        "YSVALLEUVALVALASPLYSPROALALEUARGMETILE"
-                                        "SERGLUCYSALAARGMETSERGLUILELEUASPLEUGL"
-                                        "YVALTHRVALVALGLUASPVALSERLYSGLNARGLYSV"
-                                        "ALLEUPROGLNPHEHISGLYVALTYRPHEILEGLUPRO"
-                                        "THRGLUGLUASNLEUASPTYRVALILEARGASPPHEAL"
-                                        "AASPARGTHRPROTHRTYRGLUALAALAHISLEU")
+        self.check_msms(
+            "PDB/2XHE.pdb",
+            "HISMETSERLEULYSSERALAVALLYSTHRVALLEUTH"
+            "RASNSERLEUARGSERVALALAASPGLYGLYASPTRPL"
+            "YSVALLEUVALVALASPLYSPROALALEUARGMETILE"
+            "SERGLUCYSALAARGMETSERGLUILELEUASPLEUGL"
+            "YVALTHRVALVALGLUASPVALSERLYSGLNARGLYSV"
+            "ALLEUPROGLNPHEHISGLYVALTYRPHEILEGLUPRO"
+            "THRGLUGLUASNLEUASPTYRVALILEARGASPPHEAL"
+            "AASPARGTHRPROTHRTYRGLUALAALAHISLEU",
+        )
```

Comment:
Is there any reason for these line breaks at 38 residues? Could use fewer longer lines otherwise.

Reply:
OK, now breaks at 74 residues - maximum to fit black's line length 88.

(I would have used 60 or 80 if that fit as those are relatively common in FASTA files - but this is fine)